pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -195,15 +201,18 @@ public class ServiceConfiguration implements PulsarConfiguration {
 
     @FieldContext(
             category = CATEGORY_TLS,
-            doc = "Specify the tls protocols the proxy's web service will use to negotiate during TLS Handshake.\n\n"
-                    + "Example:- [TLSv1.3, TLSv1.2]"
+            doc = """
+                    Specify the tls protocols the proxy's web service will use to negotiate during TLS Handshake.
+
+                    Example:- [TLSv1.3, TLSv1.2]"""
     )
     private Set<String> webServiceTlsProtocols = new TreeSet<>();
 
     @FieldContext(
             category = CATEGORY_TLS,
-            doc = "Specify the tls cipher the proxy's web service will use to negotiate during TLS Handshake.\n\n"
-                    + "Example:- [TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]"
+            doc = """
+                    Specify the tls cipher the proxy's web service will use to negotiate during TLS Handshake.
+                    Example:- [TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]"""
```

Comment:
Both sections have `\n\n` but you translate the first into one `\n` while the section into two `\n`s. Either works but please keep concistent.

Reply:
Ok.